packages/devextreme-cli/src/applications/application.vue.js

```diff
@@ -56,7 +56,7 @@ const preparePackageJsonForTemplate = (appPath, appName, version) => {
 async function createVueApp(name, templateOptions) {
     const { version } = templateOptions;
     const toolingVersion = extractToolingVersion(templateOptions);
-    const argList = ['-p', `@vue/cli${toolingVersion}`, 'vue', 'create', name];
+    const argList = ['-p', `@vue/cli${toolingVersion}`, 'vue', 'create', name, '--registry', 'https://registry.npmjs.org/'];
```

Comment:
Are we certain that we want to discontinue support for customers to install packages from their local npm repositories?

Reply:
Please @iBat reply

Reply:
I have found no other way to disable the use of the taobao registry. The "useTaobaoRegistry" option doesn't seem to work.

Reply:
Maybe we could turn on this only for testing?